server/src/test/java/org/elasticsearch/index/shard/IndexShardTests.java

```diff
@@ -2020,7 +2100,7 @@ public IndexSearcher wrap(IndexSearcher searcher) throws EngineException {
                 ShardRoutingHelper.initWithSameId(shard.routingEntry(), RecoverySource.StoreRecoverySource.EXISTING_STORE_INSTANCE),
                 shard.shardPath(),
                 shard.indexSettings().getIndexMetaData(),
-                wrapper,
+                null, wrapper,
```

Comment:
Nit: This is probably better on a separate line like the rest of the parameters.